codeflare-stack/README.md

```diff
@@ -0,0 +1,24 @@
+# CodeFlare-Stack
+
+Artifacts for installing the CodeFlare stack as part of ODH
+
+## Overview
+
+CodeFlare is a simple, user-friendly abstraction for scaling,
+queuing and resource management of distributed AI/ML and Python workloads.
+It consists of three components:
+
+* [CodeFlare SDK](https://github.com/project-codeflare/codeflare-sdk) to define and control remote distributed compute jobs and infrastructure with any Python based environment
+* [Multi-Cluster Application Dispatcher (MCAD)](https://github.com/project-codeflare/multi-cluster-app-dispatcher) for management of batch jobs
+* [Instascale](https://github.com/project-codeflare/instascale) for on-demand scaling of a Kubernetes cluster
```

Comment:
Is there a way we can add an overlay/patch for resource limits specifically for `Instascale` and `mcad` controllers? I see following cpu limits, which end up throwing errors when installed with minimal KfDef.
```
resources:        
  limits:          
    cpu: '2'          
    memory: 2G       
  requests:         
    cpu: '2'          
    memory: 2G
```

Alternatively, we can add resource requirements in the pre-requisites.

Reply:
@asm582 What are the minimum resource requirements for the InstaScale and MCAD controllers?

I think it will be more straightforward to have requirements added to the doc, then to add an overly here. 

@VaishnaviHire were you testing this in a fairly resource constrained environment? Is there a "maximum resources to test" limit we should be targeting?

Reply:
I agree with adding requirements to the doc. We can then point users to the expected resource requirements.

Regarding my environment, it was a development cluster with standard 4 core nodes. Following was the status of available resources when the InstaScale and MCAD controllers were in Pending state
```
 oc adm top nodes                                          
NAME                         CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%   
xxx-xxx-master-0         939m         26%    11739Mi         78%       
xxx-xxx-master-1         1286m        36%    11847Mi         79%       
xxx-xxx-master-2         1129m        32%    9401Mi          63%       
xxx-xxx-worker-0-bnjg5   488m         13%    3935Mi          26%       
xxx-xxx-worker-0-f5jbl   614m         17%    6470Mi          43%       
xxx-xxx-worker-0-ttnj9   482m         13%    6206Mi          41% 
```